config/jobs/tarmak/tarmak-presubmits.yaml

```diff
@@ -549,6 +549,7 @@ presubmits:
         - -C
         - docs
         - clean
+        - reference_docs_generate
         - spelling
```

Comment:
This will break existing PRs until your own is merged, right?

May be better to add this as a separate job with always_run set to false for the time being, so that other PRs don't fail 😄

Reply:
We should have a CI target in the Makefile. The tasks should be added to the tarmak master first and the update this here to run target `make -C docs ci`. After that we can easily update it without testing